recupero/forms.py

```diff
@@ -82,12 +96,16 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = Factura
-        fields = ('paciente',
+        fields = ('fecha_atencion',
+                  'paciente',
                   'obra_social',
+                  'profesional',
+                  'especialidad',
                   'centro_de_salud', 'fecha_atencion',
```

Comment:
Está dos veces `fecha_atencion` y con una funciona bien

```suggestion
                  'centro_de_salud',
```